.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,11 +18,4 @@ The project is undergoing daily changes. Pull Requests will be reviewed and resp
 
 ## Related PRs
```

Comment:
The related PR section is always never used. I think we should drop it.

Reply:
Agreed. Will update in a sec.